### ISSUE TITLE
Make window nonresizable on all platforms natively

### ISSUE THREAD

**Issue by YazeedAlKhalaf** [YazeedAlKhalaf/Flutter_Installer#23]
The app window should be always `1200x720` natively. So the user can't resize it.

Platforms:
- [ ] Windows
- [ ] macOS
- [ ] Linux

**Comment by yahu1031** [comment]
@YazeedAlKhalaf we have a solution for that with a package called [bitsdojo_window](https://pub.dev/packages/bitsdojo_window).

you can use something like this

```dart
void main(){
  runApp(MyApp());
  doWhenWindowReady(() {
    appWindow.minSize = const Size(500, 500);
    appWindow.maxSize = const Size(500, 500);
    appWindow.alignment = Alignment.center;
    appWindow.title = 'Flutter Installer';
    appWindow.show();
  });
}
```
BTW docs were more clear 😉

**Comment by YazeedAlKhalaf** [comment]
@yahu1031 interesting, but I am trying to look for a native way without depending on more packages. I believe the code for it would be just a couple of lines for each platform.

**Comment by yahu1031** [comment]
@YazeedAlKhalaf yes might be but not sure about it. I have never tried till date(Native code editing for desktop).

**Comment by Clashkid155** [comment]
So it can never be resized?

**Comment by yahu1031** [comment]
Yes never @Clashkid155

**Comment by Clashkid155** [comment]
@yahu1031  I know how to do this for Linux (gtk) but don't know about the other platforms since i haven't developed for them.

**Comment by yahu1031** [comment]
Use bitsdojo_window package

**Comment by Clashkid155** [comment]
That seems better
